fathom/src/surface/elaboration/reporting.rs

```diff
@@ -444,7 +467,7 @@ impl Message {
                     }
                     MetaSource::NamedPatternType(range, _) => (range, "named pattern type"),
                     MetaSource::MatchExprType(range) => (range, "match expression type"),
-                    MetaSource::ReportedErrorType(range) => (range, "error type"), // should never appear in user-facing output
+                    MetaSource::ReportedErrorType(range) => (range, "error type"),
```

Comment:
Any reason for dropping the comment here?

Reply:
Nope. Must have been a mistake which I forgot to revert.